docs/api/rest_api/rest_api_reference/input/ez-orders.raml

```diff
@@ -75,7 +75,33 @@ displayName: Orders
                     description: Error - the user has no permission to read orders.
                 404:
                     description: Error - the order does not exist.
-
+        /shipments:
```

Comment:
Are you sure is a right place for this endpoint ?

Reply:
It's for GET /orders/order/{identifier}/shipments